app/timetables/admin.py

```diff
@@ -1,7 +1,7 @@
 from django.contrib import admin
 
 from .models import (Weekday, MealOption, Course, Meal, Timetable,
```

Comment:
@andela-kanyanwu I'm okay if you prefer that answer. But for the record, it's one of the least accepted answers.